source/content/drupal-9-configuration-management.md

```diff
@@ -9,7 +9,7 @@ reviewed: "2020-10-21"
 
 Managing configuration is an extremely important part of any team website project, but in many cases, this area of the project does not receive as much attention as it deserves. The tools for Drupal 7 do not provide complete coverage of all configuration settings, leading to inconsistencies in configuration handling and inconvenient workarounds. This has led to configuration management becoming a real thorn in the side for many projects.
 
-Pantheon supports the [Drupal 8 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal site.
+Pantheon supports the [Drupal 9 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal site.
```

Comment:
The configuration directory defaults to `/config` for Integrated Composer sites (outside the docroot).